fbw-a32nx/src/systems/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -316,7 +316,7 @@ export class EfisSymbols {
             // eslint-disable-next-line no-lone-blocks
 
             // ALTN
-            if (this.flightPlanService.hasActive) {
+            if (this.flightPlanService.hasActive && this.guidanceController.activeGeometry) {
```

Comment:
Maybe using `this.guidanceController.hasGeometryForFlightPlan(...)` is nicer here in the the other if statements below.

Reply:
You're right, didn't know this method existed. Changed it.